index.js

```diff
@@ -68,30 +68,12 @@ function responseTime (options) {
  * @private
  */
 
-function createSetHeader (options) {
-  // response time digits
-  var digits = options.digits !== undefined
-    ? options.digits
-    : 3
-
-  // header name
-  var header = options.header || 'X-Response-Time'
-
-  // display suffix
-  var suffix = options.suffix !== undefined
-    ? Boolean(options.suffix)
-    : true
-
+function createSetHeader ({ digits = 3, header = 'X-Response-Time', suffix = true } = {}) {
```

Comment:
This will require a higher minimum Node version, this is ES6 syntax. So this change can go in the next major.

Reply:
Can we merge the PR into 3.x then? I am going to try to update the syntax of the hole middleware in the coming weeks.